.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Automatically request reviews
+dask/dataframe/dask_pandas.py    @kkraus14 @jakirkham @rjzamora
```

Comment:
Ideally we'd target the dispatch methods as opposed to the pandas implementations of the dispatch methods. i.e. https://github.com/dask/dask/blob/main/dask/dataframe/methods.py#L392-L439

Reply:
Yeah my thinking was that anyone touching the dispatch methods would also change the pandas dispatch methods since that's the way that they'd see any effect. Moving the pandas dispatch registers seemed like a more natural place to separate the code.

Reply:
I think moving both the dispatch objects themselves, as well as the pandas-specific implementations, into a `dask/dataframe/dispatch.py` module seems reasonable from an organizational perspective as it allows us to isolate "these are the things other DataFrame libraries can customize" to a single location. What do you think @jsignell?

Reply:
Yeah that makes sense to me. Do the top-level functions (like `dd.concat`) then also live in that module?

Reply:
I think probably the answer should be yes because otherwise won't import order matter?

Reply:
I could be missing something, but I'm not sure why top-level functions like `dd.concat` would be impacted. IIUC we should be able to move all the `Dispatch` objects and their registrations to a separate module and things should work as they do today.

Reply:
I guess I thought Keith was asking that methods like `concat` also be moved, so that they can more easily be watched. 

As a side-note, I think we need to do a separate module for the Dispatch objects and the panda registrations to avoid circular dependencies.

Reply:
Hmm that's a bummer re: circular dependencies. I'm assuming this is because `concat_pandas` calls `concat` internally so we end up with circular imports. Would something like moving the `concat_dispatch` import to the bottom of `dask/dataframe/methods.py` help?

Reply:
I might be wrong. Let me double check.